lib/inventory/inventory_api.rb

```diff
@@ -9,13 +9,13 @@ module Hawkular::Inventory
   # Client class to interact with Hawkular Inventory
   class InventoryClient < Hawkular::BaseClient
     # Create a new Inventory Client
-    # @param entrypoint [String] base url of Hawkular-inventory - e.g
-    #   http://localhost:8080/hawkular/inventory
-    # @param credentials [Hash{String=>String}] Hash of username, password, token(optional)
-    def initialize(entrypoint = nil, credentials = {})
-      @entrypoint = entrypoint
-
-      super(entrypoint, credentials)
+    # @param hash [Hash{String=>Object}] a hash containing base url of Hawkular-inventory - e.g
+    #   entrypoint: http://localhost:8080/hawkular/inventory
+    # and another sub-hash containing the hash with username, password, token(optional)
+    def initialize(hash)
```

Comment:
This was actually a bad change for 0.2.1 as this method is already in use by clients.